CONFIGURE.md

```diff
@@ -109,6 +109,8 @@ These default choices have been made in order to optimize the default performanc
 
 When `OQS_USE_OPENSSL` is `ON`, CMake also scans the filesystem to find the minimum version of OpenSSL required by liboqs (which happens to be 1.1.1). The [OPENSSL_ROOT_DIR](https://cmake.org/cmake/help/latest/module/FindOpenSSL.html) option can be set to aid CMake in its search.
 
+By default, when `OQS_USE_OPENSSL` is `ON`, OpenSSL will be linked to liboqs at build time. This is sometimes undesirable when using liboqs from other cryptographic libraries. The separate option `OQS_DLOPEN_OPENSSL` can be used to defer loading OpenSSL at run time if it is `ON`. The default for this option is `OFF`.
```

Comment:
This is somewhat confusing (e.g., which option is by default OFF and ON?). What speaks against giving this new option it's own heading (maybe a tad deeper than OQS_USE_OPENSSL, i.e., prefixed with "###")? This also would allow directly referencing it as it'll get its own anchor. Also missing is a statement along the lines "Only available on platforms supporting `dlopen`, e.g., not on Windows" (or so).

Reply:
Split it in a separate subsection, and expanded the text.